src/reformat_libyuv.c

```diff
@@ -341,6 +341,24 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     return AVIF_RESULT_NOT_IMPLEMENTED;
 }
 
+// These defines are used to to create a NULL reference to libyuv functions that
+// did not exist prior to a particular version of libyuv.
+#if LIBYUV_VERSION < 1838
+#define I422ToRGB565Matrix NULL
+#endif
+#if LIBYUV_VERSION < 1813
+#define I422ToARGBMatrixFilter NULL
+#define I420ToARGBMatrixFilter NULL
+#define I210ToARGBMatrixFilter NULL
+#define I010ToARGBMatrixFilter NULL
+#endif
+#if LIBYUV_VERSION < 1780
+#define I410ToARGBMatrix NULL
+#endif
+#if LIBYUV_VERSION < 1756
+#define I400ToARGBMatrix NULL
+#endif
+
```

Comment:
these defines are not strictly needed and we can include them inline in the loop up table definitions. but i thought it was more readable to keep all the version specific quirks together this way.

Reply:
Nit: Add a comment to note that these are the numbers of `avifRGBFormat` and `avifPixelFormat` enum values.

Reply:
Yes, let's add `AVIF_RGB_FORMAT_COUNT` (note the name does not have `PIXEL`) and `AVIF_PIXEL_FORMAT_COUNT`.